### ISSUE TITLE
Can't save to any drive other than c:\ on Windows

### ISSUE THREAD

**Issue by BryceLTaylor** [LLK/scratch-desktop#176]
### Expected Behavior

If you choose a different drive to save to, such as a second hard drive or a usb drive, it should save successfully.

### Actual Behavior

If you choose anything other than a location on the c:\ drive it will fail to save and throw an error.

![Can'tExportThumbDrive](https://user-images.githubusercontent.com/1088291/104765921-b1a30200-5737-11eb-8a85-747bd0245c28.png)

### Steps to Reproduce

On Windows, connect to a usb thumb drive or external hard drive
Create or open a project
Choose File > Save to your computer
In the file browser that opens up, navigate to the usb drive and try to save it.
Note: an error appears (see screenshot above) that says 

Save failed:
D:\[file name].sb3
EPERM: operation not permitted, mkdir 'D:\'

### Operating System and Browser

Windows 10.  Can not reproduce on Mac.

**Comment by BryceLTaylor** [comment]
This was reported in help desk ticket 289559

**Comment by apple502j** [comment]
This is due to fix of macOS saving bug. I'm sure this can be fixed by reverting #143 only for Windows.

**Comment by BryceLTaylor** [comment]
This has also come up in the help desk ticket 319562

**Comment by cwillisf** [comment]
On 3.20.1, I was able to save in `D:\asdf` (after making that directory). I wasn't able to save in `D:\` but I think that's expected with default Windows permissions. I did get a reasonable error message:

![image](https://user-images.githubusercontent.com/7019101/116592197-928a4380-a8d4-11eb-8ae3-1581ab747013.png)

In theory you should be able to save anywhere if a) the user has permission to save in their temp directory, and b) the user has permission to save in the actual destination they choose. I think LOTS of stuff would break if the user didn't have permission to save in their temp directory, but maybe the way we find the temp directory is somehow different from convention in some situations.

TL;DR: I wonder what's special about the situation described in ticket 319562
